libmamba/src/core/prefix_data.cpp

```diff
@@ -10,8 +10,10 @@
 #include <utility>
 
 #include <reproc++/run.hpp>
+#include <fmt/ranges.h>
```

Comment:
I think this should be included before the `reproc` header (should be in alphabetical order) to fix the linter.
FYI, running `pre-commit run --all` in the source folder will modify the files accordingly to match the linter rules.

Reply:
I'm having an issue with `pre-commit run --all` locally, sorry! I'll try to fix it but so far my attempts failed.